### PR TITLE
Fix Catlin CI to validate the YAML

### PR DESCRIPTION
**Pull request by vinamra28**
# Changes

catlin was performing validate command on wrong data `catlin validate
changed-files.txt` and it should be doing `catlin validate $(cat
changed-files.txt)` so fixed that.
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: tekton/ci-workspace/jobs/tekton-catalog-catlin-lint.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ spec:
         echo '```' >> catlin.txt
 
         # performing catlin validate
-        catlin validate changed-files.txt | tee -a catlin.txt
+        catlin validate $(cat changed-files.txt) | tee -a catlin.txt
```
Comment (vdemeester) [inline tekton/ci-workspace/jobs/tekton-catalog-catlin-lint.yaml:53]: or 

```bash
catlin validate < change-files.txt | tee -a catlin.txt
```

@chmouel wdyt ?

Comment (vinamra28) [reply tekton/ci-workspace/jobs/tekton-catalog-catlin-lint.yaml:53]: @vdemeester catlin doesn't takes input like this as of now and gives the following error
```
Error: requires at least 1 path to tekton resource yaml but received none
```
Probably we can update `catlin` in a follow-up PR

Comment (vdemeester) [reply tekton/ci-workspace/jobs/tekton-catalog-catlin-lint.yaml:53]: ohhh I see 😓

Comment (chmouel) [reply tekton/ci-workspace/jobs/tekton-catalog-catlin-lint.yaml:53]: or we can do the same as : 

https://github.com/tektoncd/plumbing/pull/839/files#diff-b84a0238906120be8e62bed805b8a3aaf596c2741c671f40fde8306c2704d60eL57-L59

**Comment by chmouel** [comment]
thanks

/lgtm

File: tekton/ci-workspace/jobs/tekton-catalog-catlin-lint.yaml (hunk 1 above)
Comment (vinamra28) [reply tekton/ci-workspace/jobs/tekton-catalog-catlin-lint.yaml:53]: @chmouel catlin validate command can accept an array of args so we can run `catlin validate task/git-clone/0.2 task/black/0.1` whereas `catlin lint script` accepts only one arg at a time hence the for loop was there and with `catlin validate` I passed the whole string from the file :sweat_smile:

**Pull request merged**